Formula/faunus.rb

```diff
@@ -56,9 +56,13 @@ def install
       "-S", ".", *std_cmake_args,
       "-DPYTHON_EXECUTABLE=#{venv_root}/bin/python"
     system "make", "install"
+    mv prefix/"bin/yason.py", prefix/"bin/yason.py.unwrapped"
+    (bin/"yason.py").write_env_script prefix/"bin/yason.py.unwrapped",
+      PYTHONPATH: "#{venv_root}/lib/python#{xy}/site-packages"
```

Comment:
```suggestion
    (libexec/"bin").install bin/"yason.py"
    (bin/"yason.py").write_env_script libexec/"bin/yason.py", PYTHONPATH: "#{venv_root}/lib/python#{xy}/site-packages"
```
I don't think we want users to find `yason.py.unwrapped` in their `PATH`.

Reply:
The reason is the following nastiness in `yason.py` that uses the path to locate the schema file. Do you have suggestions for an alternative location?
~~~ py
pathname = os.path.dirname(sys.argv[0]) # location of yason.py
for subdir in ["/../docs", "/../share/faunus"]: # schema file can be in src or installed
    schemafile = os.path.abspath(pathname+subdir) + '/schema.yml'
~~~

Reply:
(as I mentioned above, I'm looking for a solution that doesn't modify the source)

Reply:
Can we install everything into `libexec`, and just use `write_env_script` at the end into `bin`?

Reply:
The relative path would fit the bill if we place `yason.py` directly in `libexec`. Is this what you suggest?

Reply:
Yes. It seems to look for things using relative paths, so put everything in `libexec`. I assume users are interested only in `yason.py`, so we write an env script in `bin` for them to have access to it. If there's more in `libexec` that a user might need, then we can use `install_symlink`.

Reply:
done